tests/testthat/test-pstr_add_transition_risk.R

```diff
@@ -1,10 +1,56 @@
-test_that("hasn't change", {
-  with_transition_risk <- pstr_companies |>
-    pstr_add_reductions(pstr_ep_weo, pstr_weo_2022) |>
-    pstr_add_transition_risk() |>
-    # FIXME: Address warning
-    suppressWarnings()
-
-  out <- format_robust_snapshot(with_transition_risk)
-  expect_snapshot(out)
```

Comment:
Same here. It's best to now remove the snapshot test so the test-coverage report is more useful. If we eventually don't cover a line of code we can focus our attention there.